software/apps/mandel-full/main.c

```diff
@@ -0,0 +1,210 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "hardware/clocks.h"
+#include "hardware/dma.h"
+#include "hardware/gpio.h"
+#include "hardware/irq.h"
+#include "hardware/pll.h"
+#include "hardware/sync.h"
+#include "hardware/structs/bus_ctrl.h"
+#include "hardware/structs/ssi.h"
+#include "hardware/vreg.h"
+#include "pico/multicore.h"
+#include "pico/sem.h"
+#include "pico/stdlib.h"
+
+#include "tmds_encode.h"
+
+#include "dvi.h"
+#include "dvi_serialiser.h"
+#include "common_dvi_pin_configs.h"
+
+#include "mandelbrot.h"
+
+// TMDS bit clock 252 MHz
+// DVDD 1.2V (1.1V seems ok too)
+#define FRAME_WIDTH 640
+#define FRAME_HEIGHT 480
+#define VREG_VSEL VREG_VOLTAGE_1_10
+#define DVI_TIMING dvi_timing_640x480p_60hz
+
+#define LED_PIN 25
```

Comment:
Maybe better to use PICO_DEFAULT_LED_PIN (it's not pin 25 on my DVI boards for example), but happy for it to be left as is until I sort out proper board headers

Reply:
Cool, yep I'll fix that too.